compiler/lib/src/main/scala/codegen/CppWriter/ArrayCppWriter.scala

```diff
@@ -255,7 +261,8 @@ case class ArrayCppWriter (
           indexIterator(lines("this->elements[index] = obj.elements[index];")),
         )
       ),
-    )
+    ) ++
+      singleElementConstructor
```

Comment:
Can we put the single element constructor back where it was, before the multiple element constructor? It will cause less disruption in the unit test files. I think you can write List(...) ++ singleElementConstructor ++ List(...) instead of List(...) ++ singleElementConstructor.

Reply:
Ok, fixed!